docs/goals.md

```diff
@@ -12,43 +12,43 @@ of some data value or values.
 
 The design goals are listed below.
 
-1.  Express grammatical features, such as plurals, genders, and inflections.
+1.  Express grammatical features, such as plurals and genders.
 
 2.  Express other variance in translation, due to linguistic and regional
     features, the presentation media, context, circumstance, and other factors.
+    _Note: not part of the LDML45 release._
```

Comment:
I'm not sure that we aren't providing this. Admittedly we aren't providing explicit support for e.g. display context, but one can use `:string` (_SelectFormat_) to do most of these.

To be honest, I'm not sure why this isn't just part of the first item.

Reply:
It is quite broad: for example, "linguistic" could reasonably include gender, other noun class, inflections, general agreement. And those are problematic because the tooling needs to do expansion/contraction. Unclear whether other items also need to be "enums" that expand/contract based on locale (context? Does that include formality/linguistic register?). Anyway, this list is far to misleading.

If you really mean "stuff that can be selected with :string (SelectFormat)", then that should be explicit so that this line doesn't mislead people.

Reply:
Or just delete the line. We aren't listing all of the design features anyway.

Reply:
Deleting to resolve conversation.

Reply:
I think, actually, that this item and the first item might be better expressed as:

```suggestion
1. Allow users to write messages that are both grammatically
   correct and can be translated in a grammatically correct manner
   to languages that have different requirements. For example, plurals.
```